src/pages/user-profile-view/user-profile-view.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { clone } from 'lodash'
+import Auth0LoginZone from '../../components/auth0-login-zone'
+import ReactTable from 'react-table'
+
+import './user-profile-view.scss'
+import LoadingElement from '../../components/generic-elements/loading-element'
+import ErrorComponent from '../../components/generic-elements/error-component'
+
+const UserProfileView = (props) => {
+  const {
+    visualizations,
+    getUserVisualizations,
+    auth: { isAuthenticated },
+    loading,
+    loadFailure,
+    loadSuccess
+  } = props
+
+  useEffect(() => {
+    getUserVisualizations()
+  }, [])
+
+
+  const requestHasNotRun = !loadFailure && !loadSuccess
+  if (loading || requestHasNotRun) {
+    return (
+      <user-profile-view>
+        <LoadingElement />
+      </user-profile-view>
+    )
+  }
+
+  if (!isAuthenticated ) {
+    return <ErrorComponent errorText={"You must be signed in to see your saved visualizations"} />
+  }
+
+
+  const data = clone(visualizations)
+  data.forEach((visualization) => {
```

Comment:
Hmmm, I'm trying to think of a better way to do this than cloning. Maybe:
```suggestion
  const visualizationsWithTitle = visualizations.map((visualization) => {
  return { visualization, ...{title: (<Link to={`/visualization/${visualization.id}`}>{visualization.title}</Link>)} }
  })
```

Reply:
yeah that's better was pretty unsure about that anyway